src/database/migrations/0010-insert-canonical_fields.sql

```diff
@@ -0,0 +1,285 @@
+INSERT INTO canonical_fields (id, label, short_code, data_type) OVERRIDING SYSTEM VALUE VALUES
+(1, 'Organization Name', 'organization_name', '{ type: "string" }' ),
```

Comment:
What is this `'{ type: "string" }'` value? It looks vaguely JSON-ish, but like JavaScript notation for JSON - it is not valid JSON:

```sh
$ echo '{ type: "string" }' | jq .
parse error: Invalid literal at line 1, column 7
```

The `canonical_fiels.data_type` database field is declared as `character varying`, rather than `jsonb`, and I don't see any parsing code in the application - it looks like the API just passes the value in the database back to the HTTP client.

Reply:
It is intended to be a JSON schema type, but yeah, it looks like we need to shore it up one way or another.

Reply:
I updated the origin spreadsheet but that does not address the validation at post or at insert time.